### DISCUSSION TITLE
Advanced Melee Turtle

### DISCUSSION THREAD

**Discussion by DramoraFramei** [cc-tweaked/CC-Tweaked#1982] [Q&A]
Hey, I'm using Forge 1.19.2 with all CC: Tweak addons and was wondering if it was possible to use the gps in combination with Ducky Peripherals Entity Detector to make Advanced Melee Turtles travel along a perimeter until a mob enters said perimeter at which point it'll move to attack the mob.

**Comment by Lupus590** [comment]
You can get the turtle to swap peripherals with the turtle's equip commands.
Turtles can only have two peripherals (or tools) equipped at once, one per side. But they can carry more in their inventory and swap them around.
